packages/components/vitest.config.ts

```diff
@@ -0,0 +1,17 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
```

Comment:
Why split it?

Reply:
separation of concerns, to avoid bloating the vite config with testing setup responsibilities.